client/components/LeftArrow.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styles from '../styles/LeftArrow.css';
```

Comment:
Good that you have CSS file dedicated to JSX file. I just have 1 CSS file for all, hehe.